subprojects/ivy/src/integTest/groovy/org/gradle/api/publish/ivy/IvyPublishJavaIntegTest.groovy

```diff
@@ -558,7 +551,7 @@ class IvyPublishJavaIntegTest extends AbstractIvyPublishIntegTest {
         }
 
         and:
-        resolveArtifacts(javaLibrary)*/
+        resolveArtifacts(javaLibrary) == ['commons-collections-3.2.2.jar', 'publishTest-1.9.jar']
```

Comment:
:o: We could add the equivalent part to the maven test here, to document the expected behavior: 
- Run without experimental (i.e. without Gradle metadata)
- `failure.assertHasDescription("Could not resolve all files for configuration ':resolve'")`
- `failure.assertHasCause("Could not find commons-collections:commons-collections:")`

See: https://github.com/gradle/gradle/pull/3672/commits/57e25f7bac5f68700849f90512b8d8f8dfed949f#diff-00c096f4d1b67345691179afbdc0c14d